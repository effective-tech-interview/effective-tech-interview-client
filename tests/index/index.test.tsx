// eslint-disable-next-line import/no-unresolved
import Home from '@pages/index';
import { render, screen } from '@testing-library/react';

describe('<Home />', () => {
  it('renders a heading', () => {
    const { container } = render(<Home />);

    const home = screen.getByText('Home');

    expect(home).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
