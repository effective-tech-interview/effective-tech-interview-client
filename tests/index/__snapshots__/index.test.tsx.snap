// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders a heading 1`] = `
<div>
  <div
    style="display: flex; flex-direction: column;"
  >
    <div>
      Home
    </div>
    <span
      class="css-3j559g ek7smyr0"
      color="gray800"
      size="1.6"
    >
      이팩티브 기술면접
    </span>
    <span
      class="css-w2f84g ek7smyr0"
      color="gray800"
      size="1.6"
    >
      이팩티브 기술면접
    </span>
    <span
      class="css-uqu3dd ek7smyr0"
      color="gray800"
      size="1.6"
    >
      이팩티브 기술면접
    </span>
    <span
      class="css-vzwkk1 ek7smyr0"
      color="gray800"
      size="1.6"
    >
      이팩티브 기술면접
    </span>
    <span
      class="css-8ki022 ek7smyr0"
      color="gray800"
      size="1.6"
    >
      이팩티브 기술면접
    </span>
    <span
      class="css-9kepj9 ek7smyr0"
      color="gray800"
      size="1.6"
    >
      이팩티브 기술면접
    </span>
    <div
      style="display: flex; height: 500px; flex-direction: column; width: 328px; justify-content: space-between;"
    >
      <button
        class="css-1mmamjn enf1cef0"
        color="primary_default"
        width="7.2"
      >
        변경
      </button>
      <button
        class="css-pfqsme enf1cef0"
      >
        버튼
      </button>
      <button
        class="css-1rgmv7y enf1cef0"
      >
        버튼
      </button>
      <button
        class="css-1l939tl enf1cef0"
      >
        버튼
      </button>
      <button
        class="css-zrrv6e enf1cef0"
      >
        버튼
      </button>
      <button
        class="css-1punggx enf1cef0"
        color="gray400"
      >
        버튼
      </button>
      <button
        class="css-1c0tl3d enf1cef0"
      >
        버튼
      </button>
    </div>
    <div
      class="css-14bpej7 e16bg56v0"
    >
      <label>
        <span
          class="css-8ki022 ek7smyr0"
          color="gray800"
          size="1.6"
        />
      </label>
      <div
        class="css-17u2oxq e16bg56v1"
      >
        <input
          class="css-fsrgdv e16bg56v2"
          type="text"
        />
      </div>
    </div>
  </div>
</div>
`;
